py/py3/basic.py

```diff
@@ -9,7 +9,10 @@
 import sys
 import time
 
+sys.path.insert(0, "./wrappers")
+
 from ocutils import OCUtils
+from wrappers.report import SortDirection
```

Comment:
I think ideally we should find a way to work around working with anything in wrappers directly, outside of occampy. Some way to eventually abstract this functionality outside of these files would be good.

Reply:
It will be that way when we create the other classes